---
title: "**Data**"
output:
  html_document:
    toc: true
    toc_depth: 3
    toc_float:
      collapsed: false
      smooth_scroll: true
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

</br>

### Sequence data

All sequence data generated within this study can be accessed via the BioProject accession number [PRJNA811776](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA811776). 
</br>

<ul>
  <li>16S data are available at the Sequence Read Archive under the submission number [SUB11166467](https://submit.ncbi.nlm.nih.gov/subs/sra/SUB11166467)</li>
  <li>ITS data are available at the Sequence Read Archive under the submission number [SUB11166581](https://submit.ncbi.nlm.nih.gov/subs/sra/SUB11166581)</li>
</ul>

</br>

### Metadata

```{r metadata, echo=F, warning=F, message=F}
library(readxl)
library(reactable)
library(htmltools)
library(tidyverse)

metadata <- read_xlsx("data/metadata.xlsx", sheet = 2) %>% 
  mutate(Collection_date = format(as.Date(Collection_date, format = "%Y-%m-%d"), "%Y-%m-%d"),
         Date = format(as.Date(Date, format="%Y-%m-%d"),"%Y-%m-%d"))

htmltools::browsable(
  tagList(
    div(
      div(tags$label("Location", `for` = "location-filter")),
      tags$select(
        id = "location-filter",
        onchange = "Reactable.setFilter('ehil-metadata', 'Location', this.value)",
        tags$option("All", value = ""),
        lapply(unique(metadata$Location), tags$option)
        )
      ),
    tags$hr("aria-hidden" = "false"),
    tags$button("Download as CSV", onclick = "Reactable.downloadDataCSV('ehil-metadata', 'metadata.csv')"),
    
    reactable(
      metadata,
      columns = list(
        SampleID = colDef(minWidth = 130),
        ID = colDef(minWidth = 40, align = 'right'),
        EHIL_plate = colDef(align = 'right'),
        Control_Treatment = colDef(minWidth = 140),
        Collection_date = colDef(minWidth = 120)),
      compact = TRUE,
      searchable = T,
      defaultPageSize = 15,
      showPageSizeOptions = T,
      elementId = 'ehil-metadata'
      )
  )
)

```

</br>

### Abundance and taxonomy

The generated ASV tables and assigned taxonomy can be downloaded here.

```{r ASV tables, echo=F, warning=F, message=F}
tables <- data.frame(Processing = c('Raw output', 
                                    'Filtered output', 
                                    'Subsampled output', 
                                    'Taxonomy'), 
                     Bacteria = rep('Download', 4),
                     Bac_links = c('data/16S-seqtab.xlsx', 
                                   'https://www.google.com', 
                                   'https://www.google.com', 
                                   'data/16S-taxa.xlsx'),
                     Fungi = rep('Download', 4),
                     Fun_links = c('data/ITS-seqtab.xlsx', 
                                   'https://www.google.com', 
                                   'https://www.google.com', 
                                   'data/ITS-taxa.xlsx'),
                     Description = c('Raw output from DADA2 analysis',
                                     'All ASVs without Genus-level taxonomy removed',
                                     'Abundance rarefied to lowest sequencing depth',
                                     'Taxonomy based by SILVA v.132 or UNITE 2021-05-10'))

reactable(
  tables,
  columns = list(
    Processing = colDef(maxWidth = 170),
    Bacteria = colDef(maxWidth = 100,
                      cell = function(value, index) {
      url <- sprintf(tables[index, "Bac_links"], value)
      htmltools::tags$a(href = url, target = "_blank", as.character(value))
    }),
    Bac_links = colDef(show = F),
    Fungi = colDef(maxWidth = 100,
                   cell = function(value, index) {
      url <- sprintf(tables[index, "Fun_links"], value)
      htmltools::tags$a(href = url, target = "_blank", as.character(value))
    }),
    Fun_links = colDef(show = F)
  )
)
```

