<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Bacteria</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/simplex.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}

.tocify-subheader {
  display: inline;
}
.tocify-subheader .tocify-item {
  font-size: 0.95em;
}

</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">EHIL</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">About</a>
</li>
<li>
  <a href="data.html">Data</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Results
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="bacteria.html">Bacteria</a>
    </li>
    <li>
      <a href="fungi.html">Fungi</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:TBA@outlook.com">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore"><strong>Bacteria</strong></h1>

</div>


<p><br/></p>
<div id="data-preparation" class="section level1 tabset">
<h1 class="tabset">Data preparation</h1>
<div id="unfiltered" class="section level2">
<h2>Unfiltered</h2>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-rarecurve_unfiltered.png"> <img src="output/16S-rarecurve_unfiltered.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 1</b> - Rarefaction curve of unfiltered data.
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
<div id="filtered" class="section level2">
<h2>Filtered</h2>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-rarecurve_filtered.png"> <img src="output/16S-rarecurve_filtered.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 2</b> - Rarefaction curve of filtered data. The filter was set to in include ASVs with at least 2 reads in at least 10% of all samples.
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
<div id="rarefied-smallest-sample" class="section level2">
<h2>Rarefied (smallest sample)</h2>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-rarecurve_filtr1.png"> <img src="output/16S-rarecurve_filtr1.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 3</b> - Rarefaction curve of filtered and rarefied data. The data were rarefied to the smallest sample size.
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
<div id="rarefied-outlier-removed" class="section level2">
<h2>Rarefied (outlier removed)</h2>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-rarecurve_filtr2.png"> <img src="output/16S-rarecurve_filtr2.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 4</b> - Rarefaction curve of filtered and rarefied data. The data were rarefied to the second smallest sample size, thereby exluding the smallest sample as an outlier.
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
</div>
<div id="section" class="section level1 unlisted unnumbered">
<h1 class="unlisted unnumbered"></h1>
<p><br/></p>
</div>
<div id="community-composition" class="section level1">
<h1>Community composition</h1>
<div id="stams" class="section level2 tabset">
<h2 class="tabset">Stams</h2>
<div id="single-samples-full-dataset" class="section level3">
<h3>Single samples (full dataset)</h3>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-phylum_stams.png"> <img src="output/16S-phylum_stams.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 5</b> - Community composition at phylum level for each sample of the full dataset including the outlier J3S11 (J3_S_15_R3_S4).
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
<div id="grouped-samples-full-dataset" class="section level3">
<h3>Grouped samples (full dataset)</h3>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-phylum_stams_mean.png"> <img src="output/16S-phylum_stams_mean.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 6</b> - Average community composition at phylum level for samples grouped at sampling time point (Full dataset including the outlier J3S11 (J3_S_15_R3_S4)).
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
<div id="single-samples-outlier-removed" class="section level3">
<h3>Single samples (outlier removed)</h3>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-phylum_stams_sub.png"> <img src="output/16S-phylum_stams_sub.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 7</b> - Community composition at phylum level for each sample of the dataset without outlier J3S11 (J3_S_15_R3_S4).
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
<div id="grouped-samples-outlier-removed" class="section level3">
<h3>Grouped samples (outlier removed)</h3>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-phylum_stams_sub_mean.png"> <img src="output/16S-phylum_stams_sub_mean.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 8</b> - Average community composition at phylum level for samples grouped at sampling time point (Dataset without outlier J3S11 (J3_S_15_R3_S4)).
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
</div>
</div>
<div id="section-1" class="section level1 unlisted unnumbered">
<h1 class="unlisted unnumbered"></h1>
<p><br/></p>
<div id="pill" class="section level2 tabset">
<h2 class="tabset">Pill</h2>
<div id="single-samples" class="section level3">
<h3>Single samples</h3>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-phylum_pill.png"> <img src="output/16S-phylum_pill.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 9</b> - Community composition at phylum level for samples collected in Pill.
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
<div id="grouped-samples" class="section level3">
<h3>Grouped samples</h3>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-phylum_pill_mean.png"> <img src="output/16S-phylum_pill_mean.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 10</b> - Average community composition at phylum level for samples collected in Pill grouped at sampling time point.
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
</div>
</div>
<div id="section-2" class="section level1 unlisted unnumbered">
<h1 class="unlisted unnumbered"></h1>
<p><br/></p>
</div>
<div id="alpha-diversity" class="section level1 tabset">
<h1 class="tabset">Alpha diversity</h1>
<div id="stams-full-dataset" class="section level2">
<h2>Stams (full dataset)</h2>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-alpha_stams_full.png"> <img src="output/16S-alpha_stams_full.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 11</b> - Alpha diversity (full dataset).
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
<div id="stams-outlier-removed" class="section level2">
<h2>Stams (outlier removed)</h2>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-alpha_stams_sub.png"> <img src="output/16S-alpha_stams_sub.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 12</b> - Alpha diversity (outlier removed).
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
<div id="pill-full-dataset" class="section level2">
<h2>Pill (full dataset)</h2>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-alpha_pill.png"> <img src="output/16S-alpha_pill.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 13</b> - Alpha diversity.
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
</div>
<div id="section-3" class="section level1 unlisted unnumbered">
<h1 class="unlisted unnumbered"></h1>
<p><br/></p>
</div>
<div id="venn-diagrams" class="section level1 tabset">
<h1 class="tabset">Venn diagrams</h1>
<div id="stams-1" class="section level2">
<h2>Stams</h2>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-venn-stams.png"> <img src="output/16S-venn-stams.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 14</b> - Venn diagram showing shared ASVs across treatment groups before and after the experiment.
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
<div id="pill-1" class="section level2">
<h2>Pill</h2>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-venn-pill.png"> <img src="output/16S-venn-pill.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 15</b> - Venn diagram showing shared ASVs across treatment groups before and after the experiment.
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
</div>
<div id="section-4" class="section level1 unlisted unnumbered">
<h1 class="unlisted unnumbered"></h1>
<p><br/></p>
</div>
<div id="differential-abundance" class="section level1 tabset">
<h1 class="tabset">Differential abundance</h1>
<div id="stams-2" class="section level2">
<h2>Stams</h2>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-lefse-stams.png"> <img src="output/16S-lefse-stams.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 16</b> - Linear discriminant analysis of effect size for samples each treatment in Stams (before/after).
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
<div id="pill-2" class="section level2">
<h2>Pill</h2>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-lefse-pill.png"> <img src="output/16S-lefse-pill.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 17</b> - Linear discriminant analysis of effect size for samples each treatment in Pill (before/after).
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
</div>
<div id="section-5" class="section level1 unlisted unnumbered">
<h1 class="unlisted unnumbered"></h1>
<p><br/></p>
</div>
<div id="ordination" class="section level1 tabset">
<h1 class="tabset">Ordination</h1>
<div id="stams-3" class="section level2">
<h2>Stams</h2>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-nmds-stams.png"> <img src="output/16S-nmds-stams.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 18</b> - Non-metric multidimensional scaling for samples collected in Stams.
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
<div id="pill-3" class="section level2">
<h2>Pill</h2>
<center>
<p><a href="https://raw.githubusercontent.com/tklammsteiner/ehil/main/docs/output/16S-nmds-pill.png"> <img src="output/16S-nmds-pill.png" width="100%" /></a></p>
<figcaption align="center">
<b>Fig. 19</b> - Non-metric multidimensional scaling for samples collected in Pill.
</figcaption>
<p>
<em>click figure to see larger version</em>
</p>
</center>
</div>
</div>
<div id="section-6" class="section level1 unlisted unnumbered">
<h1 class="unlisted unnumbered"></h1>
<p><br/></p>
</div>
<div id="permanova" class="section level1 tabset">
<h1 class="tabset">PERMANOVA</h1>
<div id="stams-4" class="section level2">
<h2>Stams</h2>
<pre><code>## Permutation: free
## Number of permutations: 1000
## 
## Terms added sequentially (first to last)
## 
##            Df SumsOfSqs MeanSqs F.Model      R2   Pr(&gt;F)    
## Treatment   3   0.31042 0.10347  1.1492 0.10175 0.124875    
## Time_point  1   0.39928 0.39928  4.4347 0.13088 0.000999 ***
## Residuals  26   2.34096 0.09004         0.76736             
## Total      30   3.05066                 1.00000             
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
</div>
<div id="pill-4" class="section level2">
<h2>Pill</h2>
<pre><code>## Permutation: free
## Number of permutations: 1000
## 
## Terms added sequentially (first to last)
## 
##            Df SumsOfSqs MeanSqs F.Model      R2   Pr(&gt;F)    
## Treatment   3    0.3764 0.12547  1.0869 0.04321 0.287712    
## Time_point  1    1.5245 1.52447 13.2054 0.17498 0.000999 ***
## Residuals  59    6.8112 0.11544         0.78181             
## Total      63    8.7120                 1.00000             
## ---
## Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
</div>
</div>
<div id="section-7" class="section level1 unlisted unnumbered">
<h1 class="unlisted unnumbered"></h1>
<p><br/></p>
</div>
<div id="pairwise-permanova" class="section level1 tabset">
<h1 class="tabset">Pairwise PERMANOVA</h1>
<div id="stams-5" class="section level2">
<h2>Stams</h2>
<pre><code>## 
##  Pairwise comparisons using permutation MANOVAs on a distance matrix 
## 
## data:  vegdist(abund_tab_stams, method = &quot;bray&quot;) by group_tab_stams$Treatment_Time
## 1000 permutations 
## 
##                 0.05_End 0.05_Start 0.15_End 0.15_Start 0_End 0_Start
## 0.05_Start      0.62     -          -        -          -     -      
## 0.15_End        1.00     1.00       -        -          -     -      
## 0.15_Start      0.87     0.92       1.00     -          -     -      
## 0_End           1.00     0.95       1.00     0.84       -     -      
## 0_Start         0.78     0.64       0.78     1.00       0.73  -      
## anaerobic_End   1.00     0.98       1.00     0.92       1.00  1.00   
## anaerobic_Start 0.73     0.59       0.81     1.00       0.81  0.70   
##                 anaerobic_End
## 0.05_Start      -            
## 0.15_End        -            
## 0.15_Start      -            
## 0_End           -            
## 0_Start         -            
## anaerobic_End   -            
## anaerobic_Start 1.00         
## 
## P value adjustment method: bonferroni</code></pre>
</div>
<div id="pill-5" class="section level2">
<h2>Pill</h2>
<pre><code>## 
##  Pairwise comparisons using permutation MANOVAs on a distance matrix 
## 
## data:  vegdist(abund_tab_pill, method = &quot;bray&quot;) by group_tab_pill$Treatment_Time
## 1000 permutations 
## 
##                 0.05_End 0.05_Start 0.15_End 0.15_Start 0_End 0_Start
## 0.05_Start      0.056    -          -        -          -     -      
## 0.15_End        1.000    0.056      -        -          -     -      
## 0.15_Start      0.028    1.000      0.028    -          -     -      
## 0_End           1.000    0.028      1.000    0.028      -     -      
## 0_Start         0.028    1.000      0.028    1.000      0.056 -      
## anaerobic_End   1.000    0.028      1.000    0.028      1.000 0.056  
## anaerobic_Start 0.028    0.559      0.028    1.000      0.028 1.000  
##                 anaerobic_End
## 0.05_Start      -            
## 0.15_End        -            
## 0.15_Start      -            
## 0_End           -            
## 0_Start         -            
## anaerobic_End   -            
## anaerobic_Start 0.028        
## 
## P value adjustment method: bonferroni</code></pre>
</div>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = false;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
